rtgui/thumbnail.cc

```diff
@@ -105,6 +105,47 @@ bool CPBDump(
     return true;
 }
 
+struct ColorMapper {
+    std::map<int, std::string> indexLabelMap;
+    std::map<std::string, int> labelIndexMap;
+
+    std::map<int, std::string>::iterator it;
+    ColorMapper(std::map<int, std::string> colors) {
+        for (it = colors.begin(); it != colors.end(); it++) {
+            indexLabelMap.insert({it->first, it->second});
+            labelIndexMap.insert({it->second, it->first});
+        }
+    }
```

Comment:
Is the iterator supposed to be a local variable instead? Can the for loop be a range-based for loop?